HelloSharedLib/README.md

```diff
@@ -0,0 +1,164 @@
+
+# HelloSharedLib
+
+A simple Hello World application with Java 11+ and GraalVM that shows how to create a shared library.
+
+## Quick Instructions
+
+We use [GluonFX plugin](https://docs.gluonhq.com/) to build a native image for platforms including desktop, android and iOS.
+Please follow the prerequisites as stated [here](https://docs.gluonhq.com/#_requirements).
+
+### Desktop (Linux and macOS)
+
+Build a native image as shared library using:
+
+    mvn gluonfx:sharedlib
```

Comment:
Is `sharedlib` an existing goal in the GluonFX plugin?

Reply:
It is, see https://docs.gluonhq.com/#_gluonfxsharedlib